docker/alpine/Dockerfile_alpine_latest

```diff
@@ -50,20 +50,24 @@ ENV MYCFLAGS="-O2 -std=gnu99 -m64" \
 # List of packages to be installed (proj-data omitted: 570.04 MB)
 ENV PACKAGES="\
       attr \
+      build-base \
+      make \
       bash \
       bison \
       bzip2 \
       cairo \
       fftw \
       flex \
       freetype \
+      gcc \
       gdal \
       gettext \
       geos \
       gnutls \
       libbz2 \
       libjpeg-turbo \
       libpng \
+      libpq-dev \
```

Comment:
Should the packages added to `PACKAGES` be removed from `GRASS_BUILD_PACKAGES`?

Reply:
I am testing this, but really the `docker/alpine/Dockerfile_alpine_latest` should be rewritten to the multilayer approach as used in `docker/alpine/Dockerfile`.

Reply:
> Should the packages added to `PACKAGES` be removed from `GRASS_BUILD_PACKAGES`?

Done in a994541a38b244e4b7efdf286f7739d40c2ffebd